### PR TITLE
Create current_organic_synthesis.csl

### PR DESCRIPTION
**Pull request by singhfcy**

### CODE REVIEW

**Comment by POBrien333** [comment]
You'll need to change the underscores in the filename to dashes.

File: current_organic_synthesis.csl [hunk 1]
```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+<info>
+    <title>current-organic-synthesis</title>
+    <title-short>COC</title-short>
+    <id>http://www.zotero.org/styles/current-organic-synthesis</id>
+    <link href="http://www.zotero.org/styles/current-organic-synthesis" rel="self"/>
```
Comment (POBrien333) [inline current_organic_synthesis.csl:7]: Add the template style you used here, using rel="template".

File: current_organic_synthesis.csl [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    <link href="https://benthamscience.com/journal/manuscript-preparation.php?journalID=cos#ifa" rel="documentation"/>
+    <category citation-format="numeric"/>
+    <category field="chemistry"/>
+    <issn>1570-1794</issn>
+    <eissn>1875-6271</eissn>
+    <author>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </author>
+    <contributor>
```
Comment (POBrien333) [inline current_organic_synthesis.csl:17]: remove the contributors. They're credited via the template link.

File: current_organic_synthesis.csl [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+    <title>current-organic-synthesis</title>
+    <title-short>COC</title-short>
+    <id>http://www.zotero.org/styles/current-organic-synthesis</id>
+    <link href="http://www.zotero.org/styles/current-organic-synthesis" rel="self"/>
+    <link href="https://benthamscience.com/journal/manuscript-preparation.php?journalID=cos#ifa" rel="documentation"/>
+    <category citation-format="numeric"/>
+    <category field="chemistry"/>
+    <issn>1570-1794</issn>
+    <eissn>1875-6271</eissn>
+    <author>
```
Comment (POBrien333) [inline current_organic_synthesis.csl:13]: add yourself as the author, not Julian.

**Comment by POBrien333** [comment]
The file name is still in correct. Note I mean the filename, not the title.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**